src/comp-proc/brackets_balance.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <stdbool.h>
+bool bracketsBalanced(char *toTest) 
+{
+    int i = 0;
+    int counter = 0;
+    while (toTest[i] != '\0'){
+        if (toTest[i] == '('){
+            counter++;
+        }
+        else {
+            if (toTest[i] == ')') {
+                counter--;
+            } 
+        }
```

Comment:
Можно было чуток проще.

```suggestion
        if (toTest[i] == '(') {
            counter++;
        }
        if (toTest[i] == ')') {
            counter--;
        }
```
А ещё у Вас почему-то местами поехало форматтирование (фигурная скобочка прилипает к закрывающей скобке условия `if`). Но будем считать, что я этого не видел. Зачтено.